src/chat_statistics/stats.py

```diff
@@ -0,0 +1,62 @@
+from typing import Union
+from pathlib import Path
+import json
+from src.data import DATA_DIR
+
+from collections import Counter
+from wordcloud import WordCloud
+from bidi.algorithm import get_display
+import arabic_reshaper
+from loguru import logger
+
+
+class Chatstatistics:
+
+    def __init__(self, chat_Json: Union[str, Path] ):
+        """
+        :param chat_Json: Json file of telegram chats
+        """
+        # reading Json file
+        logger.info(f'loading chat data from {chat_Json}')
+        with open(chat_Json) as f:
+            self.chat_data = json.load(f)
+        # reading stop_words
+        logger.info(f"loading stopwords from {DATA_DIR / 'Stop_words.txt'}")
+        stop_words = open(DATA_DIR / 'Stop_words.txt').readlines()
+        self.stop_words = list(map(str.strip, stop_words))
+
+
+    def generate_wordcloud(self, output_dir = Union[str, Path], width = 5000, height = 4000):

```

Comment:
Add more optional parameters for word cloud
```suggestion
    def generate_wordcloud(self, output_dir = Union[str, Path], 
                                             width = 5000, height = 4000,
                                             max_font_size: int = 250):

```